.github/actions/setup_tests/action.yml

```diff
@@ -0,0 +1,48 @@
+name: Setup Tests
+description: Sets up the testing environment by setting up Python and Node.js, caching Python packages, installing Kedro and other Python dependencies, and building the React application.
+
+inputs:
+  os:
+    description: 'Operating system'
+    required: true
+    default: 'ubuntu-latest'
+  python-version:
+    description: 'Python version'
+    required: true
```

Comment:
this should be optional considering we have a default value

Reply:
Yes, updated.